Safety/Inc/safety_controller.hpp

```diff
@@ -0,0 +1,15 @@
+#ifndef SAFETY_CONTROLLER_H
+#define SAFETY_CONTROLLER_H
+#include "stm32f0xx_hal_iwdg.h"
+#include "PPM.hpp"
+
+/*
+*  Call this function after PPM and PWM manager have been inited.
+*/
+void safety_controller_init();
+void safety_run(PWMChannel &pwm, PPMChannel &ppm);
+bool isSafetyManual();
```

Comment:
Sounds good, some were originally static but they're bought out since interchip needs to call these functions to transmit (re: Nuclear-safety-changes interchip code) into the struct. I'll def add the descriptions though